doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -0,0 +1,276 @@
+PDU into multiple segments.
+
+The lower transport layer on the receiving device reassembles the segments into a single Upper Transport PDU before passing it
+up the stack. Delivery of a segmented message is acknowledged by the lower transport layer of the receiving node, while an unsegmented message
+delivery is not acknowledged.
+However, an Upper Transport PDU that fits into one Lower Transport PDU can also be sent as a single-segment segmented message when acknowledgment
+by the lower transport layer is required.
+Set the ``send rel`` flag (see :c:struct:`bt_mesh_msg_ctx`) to use the reliable message transmission and acknowledge single-segment segmented messages.
+
+The transport layer is able to transport up to 32 segments with its SAR mechanism, with a maximum message (PDU) size of 384 octets.
+To configure message size for the Bluetooth mesh stack, use the following Kconfig options:
+
+* :kconfig:option:`CONFIG_BT_MESH_RX_SEG_MAX` to set the maximum number of segments in an incoming message.
+* :kconfig:option:`CONFIG_BT_MESH_TX_SEG_MAX` to set the maximum number of segments in an outgoing message.
+
+The Kconfig options :kconfig:option:`CONFIG_BT_MESH_TX_SEG_MSG_COUNT` and :kconfig:option:`CONFIG_BT_MESH_RX_SEG_MSG_COUNT` define how many
+outgoing and incoming segmented messages can be processed simultaneously. When more than one segmented message is sent to the same destination,
+the messages are queued and sent one at a time.
+
+Incoming and outgoing segmented messages share the same pool for allocation of their segments. This pool size is configured through the
+:kconfig:option:`CONFIG_BT_MESH_SEG_BUFS` Kconfig option.
+Both incoming and outgoing messages allocate segments at the start of the transaction.
+The outgoing segmented message releases its segments one by one as soon as they are acknowledged by the receiver, while the incoming message releases
+the segments first after the message is fully received.
+Keep this in mind when defining the size of the buffers.
+
+SAR does not impose extra overhead on the access layer payload per segment.
+
+Intervals, timers and retransmission counters
+*********************************************
+
+The current stable stack implementation allows you to configure the following SAR behavior.
+
+When sending a segmented message to a unicast address, the unacknowledged segments are repeated
+the :kconfig:option:`CONFIG_BT_MESH_TX_SEG_RETRANS_COUNT` number of times before the transmission
+is considered as failed. The same option configures a number of retransmissions to a group or virtual
+address, but the transmission always succeedes after retransmitting all segments the configured
+number of times.
+
+The timeout between each retransmission to a unicast address and a group or virtual address is
+configured by Kconfig options :kconfig:option:`CONFIG_BT_MESH_TX_SEG_RETRANS_TIMEOUT_UNICAST` and
+:kconfig:option:`CONFIG_BT_MESH_TX_SEG_RETRANS_TIMEOUT_GROUP` respectively.
```

Comment:
This was a little bit hard to follow, "unicast address and a group or virtual address". First there is "and" then "or"? Which kconfig option is used for virtual address?